apps/pod-issuer-server/src/routes/deel.ts

```diff
@@ -0,0 +1,115 @@
+import express, { Request, Response } from "express";
+import { expressjwt, Request as JWTRequest } from "express-jwt";
+import jwt from "jsonwebtoken";
+import { POD, PODEntries } from "@pcd/pod";
+import {
+  getDeelUserByEmail,
+  getPaystubPODByEmail,
+  savePaystubPOD
+} from "../stores/deel";
+
+const deel = express.Router();
+
+deel.post("/login", (req: Request, res: Response) => {
+  const inputs: { email: string; password: string } = req.body;
+  if (!inputs.email || !inputs.password) {
+    res.status(400).send("Missing query parameter");
+    return;
+  }
+
+  // In practice, get user information from database by email,
+  // Here for demo purposes, we'll allow any email with @zoo.com domain
+  if (!inputs.email.endsWith("@zoo.com")) {
+    res.status(401).send("Invalid email or password");
+    return;
+  }
+
+  // In practice, check if the encrypted password match
+  // This is just for demo purposes
+  if (!inputs.password.startsWith("zoo")) {
+    res.status(401).send("Invalid email or password");
+    return;
+  }
+
+  // Signing JWT, valid for 1 hour
+  const token = jwt.sign(
+    { email: inputs.email },
+    process.env.DEEL_EDDSA_PRIVATE_KEY!,
```

Comment:
Is this reusing the EdDSA private key for something which isn't EdDSA?
I don't think that's dangerous, but it does somewhat confuse my understanding of which parts of this logic are related to PODs and which aren't.  Maybe it makes sense to have a separate key for login.

Reply:
oops, making a separate key for login